README.md

```diff
@@ -27,7 +38,12 @@ Run end-to-end tests with:
 
 ### Build for production
 
-`yarn build` or `npm run build`
+Run:
+* `npm run build`
+
+### Publication
+
+* publishes to [npm](https://www.npmjs.com/package/europeanamediaplayer) using a [web-action](actions?query=workflow%3A%22Node.js+Package%22)
```

Comment:
For publication maybe it's good to note that people need to have access to the npm package in order to publish this

Reply:
They can publish from this repository if they have the right to merge PRs, because the web-action runs `npm publish` using a token.

Reply:
👍